app/extensions/brave/content/scripts/flashListener.js

```diff
@@ -20,18 +22,18 @@ function isAdobeLink (href) {
       }
     })
   }
-  var observer = new window.MutationObserver(function (mutations) {
+  setTimeout(() => {
+    var observer = new window.MutationObserver(function (mutations) {
+      replaceAdobeLinks()
+    })
     replaceAdobeLinks()
```

Comment:
how about do a `document.querySelectorAll('object')` at load end and only run insertFlashPlaceholders immediately if it returns nonempty? this might get rid of the delay on most sites while keeping perf good for sites that don't load flash